recipe/meta.yaml

```diff
@@ -116,14 +114,14 @@ outputs:
         - libcxx >={{ cxx_compiler_version }}  # [osx]
     test:
       commands:
-        - test -f $PREFIX/lib/libLLVM-{{ major_ver }}{{ extra }}.so     # [linux]
-        - test -f $PREFIX/lib/libLLVM-{{ major_ver }}{{ extra }}.dylib  # [osx]
+        - test -f $PREFIX/lib/libLLVM.so.{{ maj_min }}{{ extra }}       # [linux]
```

Comment:
Thus change should also have an effect on the run_export, right?

Reply:
How so? The run-export always takes the build-time version as a lower bound, so the fact that it's  now 18.1 instead of 17.0 doesn't make a difference. The ABI-compatibility promise also stays the same, i.e. minor version increments signal ABI-changes (though that hasn't happened since LLVM 11, and usually it's just major version bumps), which is covered by our `x.x` pinning.

Reply:
Makes sense! I got confused by the package name vs run_export 🤦